jaeger_client/thrift.py

```diff
@@ -117,7 +118,7 @@ def timestamp_micros(ts):
     :param ts:
     :return:
     """
-    return long(ts * 1000000)
+    return int(ts * 1000000)
```

Comment:
I've made that change, I'm unsure if the `int` behaves as expected for py3, but we can do that in a different PR.